### ISSUE TITLE
Implementing cf-abacus

### ISSUE THREAD

**Issue by findneville** [cloudfoundry-incubator/cf-abacus#861]
Hi.. 

I have been trying to implement cf-abacus and have had difficulties just going through the documentation to implement it. It seems a bit dis-joined and/or ambiguous. 

We are interested in implementing cf-abacus and would like to contribute back to the community by making the documentation unambiguous. Is there someone who can reach out to me with regard to the same? 

I can be reached via email at neville(dot)george(at)gmail(dot)com though I think it can be hashed out via some visual tools

**Comment by hsiliev** [comment]
Hey @findneville,

Docs can be improved for sure and we'll be glad to get your feedback on them. If you decide to contribute to the documentation I can propose the following steps:
1. clone the repo and wiki
2. make changes
3. discuss them in our Slack channel (linked in github readme)
4. merge changes back

Since wiki does not support PRs this can be a bit cumbersome, so other ideas are also welcome. 

WDYT?

**Comment by findneville** [comment]
That is an awesome start. I did not know about the Slack channel but I just joined it. 

It is just that I am working through the process of implementing cf-abacus and I am stuck with interpreting some of the documentation. I started at https://github.com/cloudfoundry-incubator/cf-abacus/wiki/Install-Abacus and was immediately stuck at https://github.com/cloudfoundry-incubator/cf-abacus/wiki/Install-Abacus#creating-a-billing-epoch-and-abacus-cf-bridges with questions. 

I will reach out on the Slack channel and update the docs as I get a more clear picture of the deployment.
